packages/fast-components-react-base/src/button/button.tsx

```diff
@@ -27,6 +30,7 @@ class Button extends Foundation<IButtonHandledProps & IManagedClasses<IButtonCla
             <this.tag
                 {...this.unhandledProps()}
                 className={this.generateClassNames()}
+                {...this.renderDisabled()}
```

Comment:
Totally nit, but would prefer this method be `renderDisabledAttribute` as renderDisabled reads more like you are rendering the disabled component itself rather than attributes.

Reply:
Sure makes sense.